### PR TITLE
Open shift tornjak script

### PR DESCRIPTION
**Pull request by florentinvintila**
@mrsabath @lumjjb Please review if the changes would help with the execution of the file.
Changes done:
- moves cleanup section into the upper section to clean up in case the default configuration is used (maybe would not help as the $PROJECT param can change);
- refactoring ingress information section: moved some command where it is needed (i.e. ingressHostname);
- refactoring ingress information section: commented not needed command (i.e. ingressStatus)
- refactoring ingress information section: stoped script execution in case ingress secret name could not be retrieved (i.e. ingressSecretName);

### CODE REVIEW

**Review by mrsabath — APPROVED** [review]
Minor nits, otherwise looks good to me.

File: utils/install-open-shift-tornjak.sh [hunk 1]
```diff
@@ -28,12 +28,32 @@ Where:
...
+  #oc delete project "$PROJECT" 2>/dev/null
+}
+
 POSITIONAL=()
 while [[ $# -gt 0 ]]
 do
 key="$1"
 
 case $key in
+    --clean)
```
[the remark this answers is not in the document]
Comment (mrsabath) [reply utils/install-open-shift-tornjak.sh:53]: Also, we should do the same in `utils/install-open-shift-spire.sh`

[the remark this answers is not in the document]
Comment (florentinvintila) [reply utils/install-open-shift-tornjak.sh:53]: @mrsabath I will add the same in `utils/install-open-shift-spire.sh`

[the remark this answers is not in the document]
Comment (florentinvintila) [reply utils/install-open-shift-tornjak.sh]: @mrsabath I've moved the --clean command lower to allow the -p --project param to be passed prior to the cleanup execution.

**Pull request merged**